src/pool/pool_jemalloc.c

```diff
@@ -557,6 +557,12 @@ static umf_result_t op_get_last_allocation_error(void *pool) {
     return TLS_last_allocation_error;
 }
 
+static umf_result_t op_get_name(void *pool, const char **name) {
```

Comment:
shouldn't we add some tests with that...?

Reply:
I have some draft code for improving tests in ops area .... let's do it in a separate PR